src/app/doctors/[[...slug]]/components/list/list.module.css

```
@@ -0,0 +1,4 @@
+.container > a:not(:last-child) {
+  margin-bottom: 1rem;
+  display: block;
+}
```

به جای این کار از grid استفاده کنید

```suggestion
.container {
  display: grid;
  gap: 1rem;
}
```